fabfile/add.py

```diff
@@ -66,9 +63,8 @@ def remote_server():
         print "{0} : {1}, ".format(attr, value)
     print "}\n"
     raw_input("Press Enter to Continue or Ctrl+C to Cancel")
-    with bash():
-        local("cp -rf $FAB_PATH/../templates ./templates.tmp")
-        recursive_file_modify("./templates.tmp", settings)
+    local("cp -rf ./fabfile/../templates ./templates.tmp")
```

Comment:
`./fabfile` is a sym-link, so we have to go into it and then use the relative `..` path to get to the `templates` directory in the yurt working directory. Is there a better way to do this?